### PR TITLE
Migrated test_impact_of_replace_brick_for_glustershd.py

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the [test
case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/arbiter/brick_cases/test_impact_of_replace_brick_for_glustershd.py)

Updates: #292

Fixes: #735

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
```sh
Table:
 test_impact_of_replace_brick_for_glustershd
 
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time taken (HH:MM:SS) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     arb     |     PASS    |        00:00:51       |     N/A     |
|   dist-arb  |     PASS    |        00:01:00       |     N/A     |
+-------------+-------------+-----------------------+-------------+

Framework runtime : 00 hours 02 minutes 05 seconds
```

File: tests/functional/arbiter/brick_cases/test_impact_of_replace_brick_for_glustershd.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+# disruptive;arb,dist-arb
+# TODO: nfs
+
+from tests.d_parent_test import DParentTest
+
+
+class TestCase(DParentTest):
+
+    def run_test(self, redant):
```
Comment (srijan-sivakumar) [inline tests/functional/arbiter/brick_cases/test_impact_of_replace_brick_for_glustershd.py:31]: Can you add the steps for the TC...

File: common/ops/gluster_ops/heal_ops.py [hunk 2]
```diff
@@ -374,10 +374,11 @@ def get_self_heal_daemon_pid(self, nodes: str) -> tuple:
 
         if not isinstance(nodes, list):
             nodes = [nodes]
-        cmd = r"pgrep -f glustershd | grep -v ^$$\$"
+        # cmd = r"pgrep -f glustershd | grep -v ^$$\$"
```
Comment (srijan-sivakumar) [inline common/ops/gluster_ops/heal_ops.py:377]: Is this commented code required ?

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]

File: tests/functional/arbiter/brick_cases/test_impact_of_replace_brick_for_glustershd.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/functional/arbiter/brick_cases/test_impact_of_replace_brick_for_glustershd.py:31]: Oh yeah

File: common/ops/gluster_ops/heal_ops.py (hunk 2 above)
Comment (aujjwal-redhat) [reply common/ops/gluster_ops/heal_ops.py:377]: Forgot to remove that : )

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**